### DISCUSSION TITLE
안드로이드 키보드 올라올 때 화면이 고정되지 않는 이슈

### DISCUSSION THREAD

**Discussion by senasoon** [when-where/whenwhere-client#37] [Q&A]
안드로이드 키보드 올라올 때 화면이 고정되지 않는 이슈가 있어 글올립니다!
`androidManifest.xml` 파일에서 `windowSoftInputMode`를
`adjustNothing`과 `adjustPan`으로 수정하는 시도를 해봤는데 문제가 해결되지 않았습니다.

**Comment by imironjin** [comment]
https://reactnative.dev/docs/keyboardavoidingview 참고바랍니다.

**Comment by senasoon** [comment]
React Native Modal props 중 `statusBarTranslucent={true}` 설정을 줘서 해결했습니다🙂
현재 사용하고 있는 `react-native-modal` 라이브러리의 문제인줄 알았는데 React Native에서 기본적으로 제공하는 Modal에서도 같은 문제가 발견되었고 props 설정으로 해결할 수 있었습니다. `statusBarTranslucent`는 모달이 시스템 statusbar 아래에 위치하는지 여부를 설정하는 값입니다.

https://github.com/react-native-modal/react-native-modal/issues/344#issuecomment-1061127501
https://reactnative.dev/docs/modal#statusbartranslucent-android
